akka-docs/src/main/paradox/stream/operators/Partition.md

```diff
@@ -6,11 +6,21 @@ Fan-out the stream to several streams.
 
 ## Signature
 
+@apidoc[Partition]
+
 ## Description
 
 Fan-out the stream to several streams. Each upstream element is emitted to one downstream consumer according to the
 partitioner function applied to the element.
 
+## Example
+
```

Comment:
Some friendly explanation of sample would be nice

Reply:
added